src/test/java/bio/terra/service/resourcemanagement/google/BucketResourceTest.java

```diff
@@ -127,22 +131,45 @@ public void createAndDeleteBucketTest() throws Exception {
         checkBucketDeleted(bucketResource.getName(), bucketResource.getResourceId());
     }
 
+    @Test
+    // create buckets in different regions and see if they're actually created there.
+    public void createBucketsInDifferentRegionsTest() throws Exception {
+        for (GoogleRegion region : Arrays.asList(GoogleRegion.US_CENTRAL1, GoogleRegion.US_EAST1)) {
+            String bucketName = "testbucket_bucketregionstest_" + region;
```

Comment:
It looks like the tests in this class doesn't create unique names for its buckets, which seems like a bug to me, since two users could run the test at the same time, and bucket names are global. Unless this name is make unique later on?

Reply:
I don't know if this is a problem. The other tests don't seem to make unique bucket names.